website/lib/markdown/index.js

```diff
@@ -60,4 +62,12 @@ module.exports = {
 
     return new MergeTrees([processedDocsJsonFilesTree, processedTocFiles]);
   },
+  urlsForPrember(distDir) {
+    const flatPageListJson = fs.readJsonSync(`${distDir}/toc.json`);
+    // TODO is there a way to have this list generated automatically (or exported) from the routes (`website/app/router.js`)?
```

Comment:
I think this is doable but I did not try it as the router also contains routes we wouldn't want to crawl in this way so the fixed list seemed like an easier bet for now